svelte/src/libs/translations/languages/translation to Azerbaijan

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "en",
+"label": "English",
+"translations": {
+    "store-search-placeholder":"Paket axtarışı",
+"action": {
+      "update":"Update",
+"cancel":"Ləğv et"
+    },
+"Paket": {
+      "hamısını yeniləyin":"hamısını yeniləyin",
+"cta-mövcuddur":"quraşdırın",
+"CTA-quraşdırılıb": "quraşdırılıb",
+"CTA-quraşdırma": "quraşdırma",
+"cta-silindi": "yenidən quraşdırın",
+"cta-silmə":"sil",
+"CTA-NEEDS_UPDATE": "yeniləmə",
+"cta-UPDATİNG": "yeniləmə",
+"cta-UPDATED":"Yenilənib",
+"CTA-PRUNE": "məhsul",
+"CTA-PRUNİNG": "Budama",
+      "açıq":"açıq",
+"open-in-terminal": "terminalda aç",
+"yerli paket": "yerli paket",
+"not-in-pantry": "hələ pantry-yə əlavə edilməyib"
+    },
+"altbilgi": {
+      "Quick links-adı": "Quick links",
+"çay mağazası haqqında":"çay mağazası haqqında",
+"Problemi bildirin":"Problemi bildirin",
+"veb saytını ziyarət edin": "çay ziyarət edin.xyz",
+"Xidmət Şərtləri": "şərtlər və xidmətlər",
+"məxfilik siyasəti":"Məxfilik Siyasəti"
+    },
+"sənədlər": {
+      "Başlıq": "sənədlər",
+"seminarlar": "seminarlar"
+    },
+"Hamısına bax":"Hamısına bax",
+"çeşidləmə": {
+      "etiket": "sırala",
+"Populyarlıq" :" ən populyar",
+"ən son": "ən son"
+    },
+"ümumi": {
+      "home": "Ana səhifə",
+"all":"All",
+"articles":"Məqalələr",
+"seminar": "seminarlar",
+"metadata": "metadata",
+"Ana səhifə": "Ana səhifə",
```

Comment:
Only the mapping (right side) should be changed. The key (left side) must remain in English to be looked up.